src/server/pfs/server/testing/server_test.go

```diff
@@ -352,7 +352,7 @@ func TestPFS(suite *testing.T) {
 			{tu.UniqueString("lenny-123"), ""},
 			// {tu.UniqueString("_project"), badFormatErr}, // Require CORE-1343
 			// {tu.UniqueString("project-"), badFormatErr},
-			{pfs.DefaultProjectName, "already exists"},
+			{pfs.DefaultProjectName, "duplicate key value violates unique constraint \"projects_name_key\" (SQLSTATE 23505)"},
```

Comment:
I bet the end user wants "already exists" as the message.  Internally, `src/internal/errutil/errutil.go` contains a utility that looks for that exact string 😓.